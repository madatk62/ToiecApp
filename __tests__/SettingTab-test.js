import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import SettingTab from '../App/Components/SettingTab';

test('renders correctly', () => {
    const tree = renderer.create(<SettingTab darkMode={false} />).toJSON();
    expect(tree).toMatchSnapshot();
});