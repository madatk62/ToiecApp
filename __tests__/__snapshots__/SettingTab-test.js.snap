// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "backgroundColor": "#EEEEEE",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#1976D2",
        "flexDirection": "row",
        "justifyContent": "center",
        "paddingBottom": 20,
        "paddingTop": 40,
      }
    }
  >
    <Text
      style={
        Object {
          "alignContent": "center",
          "color": "white",
          "fontSize": 20,
          "fontWeight": "bold",
        }
      }
    >
      CÀI ĐẶT
    </Text>
  </View>
  <RCTScrollView
    style={
      Object {
        "flex": 1,
        "margin": 20,
      }
    }
  >
    <View>
      <View
        style={
          Object {
            "backgroundColor": "#F5F5F5",
            "elevation": 6,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "paddingLeft": 10,
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": "#1976D2",
                  "fontSize": 40,
                },
                undefined,
                Object {
                  "fontFamily": "FontAwesome",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            
          </Text>
          <Text
            style={
              Object {
                "color": "#1976D2",
                "fontSize": 20,
                "paddingLeft": 15,
              }
            }
          >
            Nhắc nhở 
          </Text>
          <View
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "margin": 5,
                "marginLeft": 510,
                "marginRight": 10,
                "opacity": 1,
                "padding": 10,
              }
            }
            testID="checkbox"
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "flexDirection": "row",
                }
              }
            >
              <Text
                allowFontScaling={false}
                style={
                  Array [
                    Object {
                      "color": "#1976D2",
                      "fontSize": 30,
                    },
                    Object {
                      "minWidth": 30,
                    },
                    Object {
                      "fontFamily": "FontAwesome",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                    },
                    Object {},
                  ]
                }
              >
                
              </Text>
            </View>
          </View>
        </View>
        <View
          style={
            Object {
              "flexDirection": "row",
              "justifyContent": "space-between",
              "paddingLeft": 10,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#424242",
                "fontSize": 18,
              }
            }
          >
            Số lần trong ngày
          </Text>
          <View
            style={
              Object {
                "height": 20,
                "width": 75,
              }
            }
          >
            <RCTPicker
              items={
                Array [
                  Object {
                    "label": "1",
                    "textColor": 4282532418,
                    "value": "1",
                  },
                  Object {
                    "label": "2",
                    "textColor": 4282532418,
                    "value": "2",
                  },
                ]
              }
              onChange={[Function]}
              selectedIndex={0}
              style={
                Array [
                  Object {
                    "height": 216,
                  },
                  undefined,
                ]
              }
            />
          </View>
        </View>
        <View
          style={
            Object {
              "flexDirection": "row",
              "justifyContent": "space-between",
              "paddingLeft": 10,
              "paddingRight": 10,
              "paddingTop": 20,
            }
          }
        >
          <View
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#424242",
                  "fontSize": 18,
                }
              }
            >
              Thời gian bắt đầu
            </Text>
          </View>
          <Text
            style={
              Object {
                "color": "#424242",
                "fontSize": 16,
                "marginRight": 25,
              }
            }
          >
            6:00
          </Text>
          <Modal
            animationType="none"
            hardwareAccelerated={false}
            transparent={true}
            visible={false}
          >
            <View
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "backgroundColor": "black",
                  "bottom": 0,
                  "height": 1334,
                  "left": 0,
                  "opacity": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 750,
                }
              }
            />
          </Modal>
        </View>
        <View
          style={
            Object {
              "flexDirection": "row",
              "justifyContent": "space-between",
              "paddingBottom": 20,
              "paddingLeft": 10,
              "paddingRight": 10,
              "paddingTop": 20,
            }
          }
        >
          <View
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#424242",
                  "fontSize": 18,
                }
              }
            >
              Thời gian kết thúc
            </Text>
          </View>
          <Text
            style={
              Object {
                "color": "#424242",
                "fontSize": 16,
                "marginRight": 25,
              }
            }
          >
            21:00
          </Text>
          <Modal
            animationType="none"
            hardwareAccelerated={false}
            transparent={true}
            visible={false}
          >
            <View
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "backgroundColor": "black",
                  "bottom": 0,
                  "height": 1334,
                  "left": 0,
                  "opacity": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 750,
                }
              }
            />
          </Modal>
        </View>
      </View>
      <View
        style={
          Object {
            "alignContent": "center",
            "alignItems": "center",
            "backgroundColor": "#F5F5F5",
            "elevation": 6,
            "flexDirection": "row",
            "marginTop": 20,
            "padding": 10,
          }
        }
      >
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": "#1976D2",
                "fontSize": 34,
              },
              undefined,
              Object {
                "fontFamily": "Material Design Icons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
        >
          
        </Text>
        <Text
          style={
            Object {
              "color": "#1976D2",
              "fontSize": 20,
              "paddingLeft": 10,
            }
          }
        >
           Giao diện tối
        </Text>
        <RCTSwitch
          accessibilityRole="switch"
          onChange={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              Object {
                "height": 31,
                "width": 51,
              },
              Object {
                "marginLeft": 490,
              },
            ]
          }
          value={false}
        />
      </View>
      <View
        style={
          Object {
            "backgroundColor": "#F5F5F5",
            "elevation": 6,
            "marginTop": 20,
          }
        }
      >
        <View
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "opacity": 1,
            }
          }
        >
          <View
            style={
              Object {
                "alignContent": "center",
                "alignItems": "center",
                "flexDirection": "row",
                "padding": 10,
              }
            }
          >
            <Text
              allowFontScaling={false}
              style={
                Array [
                  Object {
                    "color": "#1976D2",
                    "fontSize": 40,
                  },
                  undefined,
                  Object {
                    "fontFamily": "FontAwesome",
                    "fontStyle": "normal",
                    "fontWeight": "normal",
                  },
                  Object {},
                ]
              }
            >
              
            </Text>
            <Text
              style={
                Object {
                  "color": "#1976D2",
                  "fontSize": 20,
                  "paddingLeft": 10,
                }
              }
            >
               Chia sẻ đến bạn bè
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "#F5F5F5",
            "elevation": 6,
            "marginTop": 20,
          }
        }
      >
        <View
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "opacity": 1,
            }
          }
        >
          <View
            style={
              Object {
                "alignContent": "center",
                "alignItems": "center",
                "flexDirection": "row",
                "padding": 10,
              }
            }
          >
            <Text
              allowFontScaling={false}
              style={
                Array [
                  Object {
                    "color": "#1976D2",
                    "fontSize": 34,
                  },
                  undefined,
                  Object {
                    "fontFamily": "FontAwesome",
                    "fontStyle": "normal",
                    "fontWeight": "normal",
                  },
                  Object {},
                ]
              }
            >
              
            </Text>
            <Text
              style={
                Object {
                  "color": "#1976D2",
                  "fontSize": 20,
                  "paddingLeft": 10,
                }
              }
            >
               Báo lỗi hoặc góp ý
            </Text>
          </View>
        </View>
      </View>
    </View>
  </RCTScrollView>
</RCTSafeAreaView>
`;
